src/Agent.Listener/Configuration/WindowsServiceControlManager.cs

```diff
@@ -37,50 +39,311 @@ public override bool ConfigureService(AgentSettings settings, Dictionary<string,
 
             if (!defaultServiceAccount.Equals(new NTAccount(logonAccount)))
             {
-                Trace.Verbose("Acquiring logon account password");
-                logonPassword = consoleWizard.ReadValue(WindowsLogonPassword,
-                                                    StringUtil.Loc("WindowsLogonPasswordDescription", logonAccount),
-                                                    true,
-                                                    string.Empty,
-                                                    //TODO find how to validate using NativeMethods.LogonUser
-                                                    Validators.NonEmptyValidator,
-                                                    args,
-                                                    enforceSupplied);
-
-                Trace.Verbose("Acquired credential for logon account");
+                while (true)
+                {
+                    Trace.Info("Acquiring logon account password");
+                    logonPassword = consoleWizard.ReadValue(WindowsLogonPassword,
```

Comment:
Fyi - i think Stan is working on a fix for the ctrl+c issue not being able to terminate the app altogether while being prompted.